app/views/helpers/line_break_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class LineBreakHelper
+  def self.break_lines(text)
+    text.to_s.gsub('\n', '<br/>').html_safe
```

Comment:
Rails/OutputSafety: Tagging a string as html safe may be a security risk.

Reply:
I think if we `html_safe` it, we also have to sanitize it before, just to be safe